Makefile.host

```diff
@@ -6,30 +6,30 @@
 # %LICENSE%
 #
 
-CROSS ?=
+CROSS :=
```

Comment:
note: not sure why, other targets allow to overwrite CROSS by using ?= i Makefile.

Reply:
IMHO this is explicit host build (not cross-build) so:
 - setting `CROSS` is forbidden (until we would want to support Canadian Cross Build, eg. building hostutils for arm-linux or Win under Linux). I had problems with non-clean env and CROSS being set when re-execing build.sh (but there are mitigated now by restoring original env).
 - the canadian build use case was not ever tested, so I explicitly disabled it for now
 - maybe allowing to supply externally `$(CC)` would make more sense (eg. using clang as a host compiler on Darwin), but I haven't tested that also so not changing anything right now

Reply:
I think mentioned use case is standard CROSS, but not matter. 
I just not really sure what is the reason for overwriting CROSS in other targets, so just place a note here.

Reply:
I vote for `$(CC)` and other externally settable, would be good on OpenBSD where clang is the default, also macOS.

Reply:
I'm all in regarding supporting other toolchains - but as it's a separate issue (and I don't really have the test setup) I would welcome separate PR on this topic :)